fireapi/model/__init__.py

```diff
@@ -39,7 +40,7 @@ class RegisteringFraObjekt(DeclarativeBase):
         default=func.sysdate(),
     )
 
-    @property
+    @hybrid_property
```

Comment:
What does this do?

Reply:
It makes it possible to use the property in sqlalchemy expressions, the standard ones does not.

Reply:
Ah, now I remember.

Originally I decided to not use `hybrid_property` because it is evaluated client side which is obviously not a good idea.

The thought was that if we need to filter on `registereringfra`, we can use the proper SQLAlchemy property `_registreringfra`.

Reply:
Hmm didn't realize that difference, I would have thought @property also was evaluated client side. Actually I don't understand how a @property can be evaluated any other way?